projects/example-app/README.md

```diff
@@ -11,11 +11,11 @@ Built with [@angular/cli](https://github.com/angular/angular-cli)
 
 ### Included
 
-- [@ngrx/store](../../docs/store/README.md) - RxJS powered state management for Angular apps, inspired by Redux
```

Comment:
Will you fix the links in the previous section also?

Reply:
Yes, no problem. While I am at it should @angular/router be directed to https://angular.io/api/router instead of current https://github.com/angular/angular ?

Reply:
Point it at https://angular.io/guide/router instead.

Reply:
Okey, all requested changes fixed.